### PR TITLE
Fix #16988 add search prefixes to doc directory search

### LINKED ISSUE #16988: Linux version doesn't have change log.
### Operating System

KDE Neon 20.04

### OpenRCT2 build

OpenRCT2 v0.3.5.1

### Describe the issue

There is no change log listed in the app image or ppa version of openrct2 release.

### Area(s) with issue?

_No response_

### Steps to reproduce

1. Download app image and ppa version of openrct2
2. Run it.
3. Click on the main logo in game
4. See regardless it has no change log.

### Attachments

https://cdn.discordapp.com/attachments/691736850980667514/964660122745978980/unknown.png
https://cdn.discordapp.com/attachments/691736850980667514/964664092273025084/unknown.png

### PR DESCRIPTION
**Pull request by csunday95**
Note: trying to figure out how to test if this fixes the AppImage, but I'm having issues building it locally
 - fix #16988 
 - built appimage was previously searching only in absolute directories,
   and when doc dir was not found, defaulted to install path
 - search login now matches search prefixes from GetInstallPath()

### CODE REVIEW

File: src/openrct2/platform/Platform.Linux.cpp [hunk 1]
```diff
@@ -166,6 +108,75 @@ namespace Platform
...
+        prefixes.push_back("/");
+        return prefixes;
+    }
+
+    std::string GetDocsPath()
+    {
+        const auto prefixes = GetPathSearchPrefixes();
+        static const utf8* searchLocations[] = {
+            "./doc",
+            "/usr/share/doc/openrct2",
```
Comment (IntelOrca) [inline src/openrct2/platform/Platform.Linux.cpp:126]: Unless you have an empty prefix in that list above, you will get `//usr/share/doc/openrct2`.

File: src/openrct2/platform/Platform.Linux.cpp [hunk 2]
```diff
@@ -166,6 +108,75 @@ namespace Platform
...
+        static const char* SearchLocations[] = {
+            "/data",
+            "../share/openrct2",
+#    ifdef ORCT2_RESOURCE_DIR
+            // defined in CMakeLists.txt
+            ORCT2_RESOURCE_DIR,
+#    endif // ORCT2_RESOURCE_DIR
+            "/usr/local/share/openrct2",
+            "/var/lib/openrct2",
+            "/usr/share/openrct2",
```
Comment (IntelOrca) [inline src/openrct2/platform/Platform.Linux.cpp:163]: Same issue here

File: src/openrct2/platform/Platform.Linux.cpp (hunk 1 above)
Comment (csunday95) [reply src/openrct2/platform/Platform.Linux.cpp]: should be addressed now

**Review by IntelOrca — CHANGES_REQUESTED** [review]
I still don't think the search paths are right.
I am not keen on working directory ever being checked, the only time I see this being useful is when developing - and even then it would be handled by exe path directory.

**Pull request closed without merging**